### PR TITLE
orca: support to parse utilization from raw text

### PR DESCRIPTION
**Pull request by wbpcode**
Commit Message: orca: support to parse utilization from raw text
Additional Description:

The current orca parser forget to parse utilization for txt format. This PR added it.

Risk Level: low.
Testing: unit.
Docs Changes: n/a.
Release Notes: n/a.
Platform Specific Features: n/a.

### CODE REVIEW

File: source/common/orca/orca_parser.cc [hunk 1]
```diff
@@ -84,6 +94,11 @@ absl::Status tryCopyMetricToOrcaLoadReport(absl::string_view metric_name,
         fmt::format("custom backend load metric value({}) cannot be infinity.", metric_name));
   }
 
+  if (absl::StartsWith(metric_name, kUtilizationPrefix)) {
+    auto metric_name_without_prefix = absl::StripPrefix(metric_name, kUtilizationPrefix);
```
Comment (adisuissa) [inline source/common/orca/orca_parser.cc:98]: nit: use explicit type (absl::string_view)

File: source/common/orca/orca_parser.cc [hunk 2]
```diff
@@ -49,6 +49,16 @@ absl::Status tryCopyNamedMetricToOrcaLoadReport(absl::string_view metric_name, d
   return absl::OkStatus();
 }
 
+absl::Status tryCopyUtilizationToOrcaLoadReport(absl::string_view metric_name, double metric_value,
+                                                OrcaLoadReport& orca_load_report) {
+  if (metric_name.empty()) {
+    return absl::InvalidArgumentError("utilization metric key is empty.");
+  }
+
+  orca_load_report.mutable_utilization()->insert({std::string(metric_name), metric_value});
```
Comment (adisuissa) [inline source/common/orca/orca_parser.cc:58]: I suggest adding a validation that the metric value is in [range](https://github.com/cncf/xds/blob/b4127c9b8d78b77423fd25169f05b7476b6ea932/xds/data/orca/v3/orca_load_report.proto#L38), and return an error if it isn't.

**Review by adisuissa — COMMENTED** [review]
Thanks, overall LGTM!
cc @blake-snyder @efimki

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Comment by wbpcode** [comment]
friendly ping for another review/approval because this change some core code in some way. cc @ggreenway

**Review by ggreenway — APPROVED** [review]

**Pull request merged**